bitnami/mariadb-galera/Chart.yaml

```
@@ -27,4 +27,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/mariadb-galera
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 7.4.15
+version: 7.4.16
```

This is something new this should be 7.5.0
```suggestion
version: 7.5.0
```